subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -1023,7 +1113,7 @@ static void cig_disabled_cb(void *param)
 
 	cig = HDR_LLL2ULL(param);
 
-	if (IS_PERIPHERAL(cig) || cig->cis_count == 0) {
```

Comment:
This should not be removed. Works together with `ll_cig_remove `to cleanup when all CISes have been terminated.